src/main/java/org/breedinginsight/brapi/v2/StudyController.java

```diff
@@ -0,0 +1,68 @@
+package org.breedinginsight.brapi.v2;
+
+import io.micronaut.http.HttpResponse;
+import io.micronaut.http.HttpStatus;
+import io.micronaut.http.MediaType;
+import io.micronaut.http.annotation.*;
+import io.micronaut.http.server.exceptions.InternalServerException;
```

Comment:
this import is unused and can be removed

Reply:
Done, 47af603e.